py3status/modules/uptime.py

```diff
@@ -94,7 +94,7 @@ def post_config_hook(self):
     def uptime(self):
         with Path("/proc/uptime").open() as f:
             up = int(float(f.readline().split()[0]))
-            offset = time() - up
+            offset = time.time() - up
```

Comment:
```suggestion
            offset = time.perf_counter() - up
```

Reply:
Not in this case. The `up` is a unix timestamp and we want to calculate the difference to it, so we have to ask for `time.time()`.